docker/server/init-defaults.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+# define defaults
+CLICKHOUSE_DEFAULT_DB="${CLICKHOUSE_DEFAULT_DB:-}";
+CLICKHOUSE_DEFAULT_USER="${CLICKHOUSE_DEFAULT_USER:-}";
```

Comment:
clickhouse have a default user defined in default configuration. He's name is default. 
I would expect that env variable to have a proper value in default case too.

Reply:
Fixed